src/Uno.UI/UI/Xaml/Media/VisualTreeHelper.cs

```diff
@@ -748,6 +731,22 @@ internal static IEnumerable<UIElement> GetManagedVisualChildren(_View view)
 		internal static MaterializableList<UIElement> GetManagedVisualChildren(_View view)
 			=> view._children;
 #endif
+
+#if __IOS__ || __MACOS__ || __ANDROID__ || IS_UNIT_TESTS
+		internal static IEnumerator<UIElement> GetManagedVisualChildrenReversedEnumerator(_View view)
+			=> GetManagedVisualChildren(view).Reverse().GetEnumerator();
+#else
+		internal static MaterializableList<UIElement>.ReverseEnumerator GetManagedVisualChildrenReversedEnumerator(_View view)
+			=> view._children.GetReverseEnumerator();
+#endif
+
+#if __IOS__ || __MACOS__ || __ANDROID__ || IS_UNIT_TESTS
+		internal static IEnumerator<UIElement> GetManagedVisualChildrenReversedEnumerator(_View view, Predicate<UIElement> predicate)
+			=> GetManagedVisualChildren(view).Where(elt => predicate(elt)).Reverse().GetEnumerator();
+#else
+		internal static MaterializableList<UIElement>.ReverseReduceEnumerator GetManagedVisualChildrenReversedEnumerator(_View view, Predicate<UIElement> predicate)
+			=> view._children.GetReverseEnumerator(predicate);
+#endif
```

Comment:
I might be missing it, but I'm not seeing where we are calling this overload.

Reply:
Right, finally I remove their usage, but as this is somehow quite common, I preferred to keep them

Reply:
Sounds good